src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -0,0 +1,53 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.impl.response.OrderResponseMapper;
+import mate.academy.spring.model.dto.request.OrderRequestDto;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import mate.academy.spring.service.OrderService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderService orderService;
+    private final UserService userService;
+    private final ShoppingCartService shoppingCartService;
+    private final OrderResponseMapper orderResponseMapper;
+
+    @Autowired
+    public OrderController(OrderService orderService,
+                           UserService userService,
+                           OrderResponseMapper orderResponseMapper,
+                           ShoppingCartService shoppingCartService) {
+        this.orderService = orderService;
+        this.userService = userService;
+        this.orderResponseMapper = orderResponseMapper;
+        this.shoppingCartService = shoppingCartService;
+    }
+
+    @PostMapping("/complete/{userId}")
```

Comment:
![image](https://user-images.githubusercontent.com/12864128/194535756-3edb9d7e-e38b-4f7a-a813-216b7072ccf0.png)
It's not pathVariable

Reply:
Done